html-report/src/components/LogContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import LogFilter from './LogFilter'
+
+export default class LogContainer extends Component {
+
+  state = {
+    logs: [],
+    results: [],
+    loading: true,
+    hide: false
+  }
+
+  componentDidMount() {
+    document.addEventListener('DOMContentLoaded', (event) => {
+      this.loadStaticLogs()
+    })
+  }
+
+  loadStaticLogs() {
+    let logDivs = document.querySelectorAll('#static_logs > div > div')
+    if (!logDivs.length) {
+      this.setState({hide: true, loading: false})
+      return
+    }
+    let logs = []
+    for (var div of logDivs) {
+      logs.push(Object.assign({level: div.className, line: div.innerText, key: logs.length}))
+    }
+    this.setState({logs: logs, results: logs, loading: false})
+    window.document.getElementById("static_logs").remove()
```

Comment:
Changing DOM directly with JS in React is a forbidden action and it is against basic React rules. Please read the React manual for more information.